models/repo.go

```diff
@@ -413,8 +413,17 @@ func (repo *Repository) innerAPIFormat(e Engine, mode AccessMode, isParent bool)
 	numReleases, _ := GetReleaseCountByRepoID(repo.ID, FindReleasesOptions{IncludeDrafts: false, IncludeTags: true})
 
 	return &api.Repository{
-		ID:                        repo.ID,
-		Owner:                     repo.Owner.APIFormat(),
+		ID: repo.ID,
+		// TODO use convert.ToUser(repo.Owner)
```

Comment:
I cant use convert, this will end in a dependency circle

Repository.APIFormat() will be moved in another Pull to convert <- this will be a huge diff itselve ...

Reply:
so we can add ``convert.ToAPIRepository()`` :)